Cargo.toml

```diff
@@ -2,7 +2,7 @@
 name = "vob"
 description = "Vector of Bits with Vec-like API and usize backing storage"
 repository = "https://github.com/softdevteam/vob/"
-version = "1.0.0"
+version = "1.0.1"
```

Comment:
I'm not sure this is right:
```

    MAJOR version when you make incompatible API changes,
    MINOR version when you add functionality in a backwards-compatible manner, and
    PATCH version when you make backwards-compatible bug fixes.
```

You are adding features that don't break existing stuff, so you should bump minor, right?

Reply:
Good point!